changelogs/unreleased/70717.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Please remove this file (2 commits seems here) and add the changelog for this ticket